src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -926,6 +935,13 @@ public function get_item_schema() {
 					'type'        => 'integer',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'modified'       => array(
+					'description' => __( "The date the template was last modified, in the site's timezone." ),
```

Comment:
```suggestion
					'description' => __( 'The date the template was last modified, in the site's timezone.' ),
```

Reply:
The double quote are there because of the single quotes in `site's`. I can update the language to avoid the apostrophe if you think it's better.

Reply:
Ohh leave it then or you can update like below.

`__( 'The date the template was last modified, in the site\'s timezone.' )`